sample/shared/src/commonMain/kotlin/com/plusmobileapps/rickandmorty/locations/LocationListItem.kt

```diff
@@ -2,8 +2,8 @@ package com.plusmobileapps.rickandmorty.locations
 
 import com.plusmobileapps.rickandmorty.api.locations.Location
 
-sealed interface LocationListItem {
-    data class LocationItem(val value: Location) : LocationListItem
-    object NextPageLoading : LocationListItem
+sealed class LocationListItem {
```

Comment:
using sealed interfaces made this weird in swift by combining the entire word in one instead of being able to use a `.`